docs/data/material/components/icons/icons.md

```diff
@@ -36,15 +36,7 @@ yarn add @mui/icons-material
 
 These components use the MUI `SvgIcon` component to render the SVG path for each icon, and so have a peer-dependency on `@mui/material`.
 
-If you aren't already using MUI in your project, you can add it with:
-
-```sh
-// with npm
-npm install @mui/material
-
-// with yarn
-yarn add @mui/material
-```
+If you aren't already using MUI in your project, you can add it following the [installation guide](https://mui.com/getting-started/installation/).
```

Comment:
So it keeps working when we move the docs to a subdomain, e.g. https://v3.mui.com/

```suggestion
If you aren't already using MUI in your project, you can add it following the [installation guide](/getting-started/installation/).
```